solutions/access-management/version.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.3.0, <1.7.0"
+  required_providers {
+    # Use "greater than or equal to" range in modules
```

Comment:
update the code comment to say: `# Lock DA into an exact provider version - renovate automation will keep it updated`

And also add the rule to renovate.json file (same as [this one](https://github.com/terraform-ibm-modules/terraform-ibm-scc-da/blob/main/renovate.json))